src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -297,64 +334,105 @@ const ReleaseContent = () => {
 
       <ReleaseHeadlines release={release} />
 
-      {(release.downloadFiles || release.hasPreReleaseAccessList) && (
-        <ReleaseDataAndFilesAccordion
-          release={release}
-          renderAllFilesButton={
-            <Button
-              disableDoubleClick
-              variant="secondary"
-              onClick={() => releaseFileService.downloadAllFilesZip(release.id)}
-            >
-              Download all data
-            </Button>
-          }
-          renderDownloadLink={file => (
-            <ButtonText
-              onClick={() =>
-                releaseDataFileService.downloadFile(
-                  release.id,
-                  file.id,
-                  file.fileName,
-                )
-              }
-            >
-              {file.name}
-            </ButtonText>
+      {(release.downloadFiles ||
+        release.hasPreReleaseAccessList ||
+        hasRelatedDashboardsSection) && (
+        <div className={styles.additionalInfoContainer}>
+          <Accordion id="additional-info-accordion" showOpenAll={false}>
```

Comment:
A more suitable id would be `data-accordion`, and this works well with the suggestion to rename `ReleaseDataAndFilesAccordion` to `ReleaseDataFilesAccordion`